### DISCUSSION TITLE
Looking for new maintainers

### DISCUSSION THREAD

**Discussion by simprecicchiani** [simprecicchiani/ThinkPad-T460s-macOS-OpenCore#104] [General]
As much as I loved my T460s, our relationship has sadly come to an end.
The increasing workload I had to deal everyday since the past few months put the poor ThinkPad under pressure. Some weeks ago, I had the opportunity to buy an M1 MBA from a firm I collaborate with and didn't say no to it.

This journey has been a first time to me. And I enjoyed it a lot.
Back in March 2020, when I first bought it, the T460s wasn't really ready for our beloved macOS. The first two month I really struggled to get something decent. OpenCore wasn't as known as it is today: guides were scarce and the community narrow.
Then, thanks to the inputs given by some guys at InsanelyMac, things were starting to look promising. I published the repo and immediately had good support from the community. This motivated me a lot to keep improving and updating regularly.
I put a lot of effort into the documentation, trying to be as clear and exhaustive as possible without making it too long. I took inspiration from different places to develop a nice a tidy layout that I saw reused in many repos. A year from then, this repo counts 150⭐️, 30 forks and more than 100 issues. This made me so proud and I hope the hackintosh community benefited from my contribution. I look forward to see further development for this project by all the enthusiasts.

You are obviously free to fork the repo and make it your own project but I think becoming new maintainer(s) of this repo will make it easier for those who already know it, since the traffic has been - and still is - huge.

**TL;DR**
I don't own this laptop anymore. Those who are willing to maintain this project can write me down here.

**Comment by 559911** [comment]
Thank you for your great work,I'v learned a lot.

**Reply by MarkMorrow** [reply]
I second that.  Thanks so much @simprecicchiani !!!  

I had noticed the lack of updates, but just figured it had reached perfection so nothing more to do.  Hopefully someone with time and knowledge will takes it up,  as Apple is sure to push out updates which will break things in the future.  For now, my T460s is working well thanks to you.  :)

**Comment by mhl221135** [comment]
Hello, it's a pity that you are no longer with us, I think I could continue your hackishntosh path) I still need to learn a lot before I can release updates as often as you do, but I think I can handle it because I really like my t460s hackintosh

**Reply by simprecicchiani** [reply]
You can surely do! Just try to test the changes by yourself first, share the preview with our community and finally, when you feel comfortable with it, release it to the public.
Updating the package is a fairly easy task, anyone who's willing to commit to it can do it. I saw you were testing things to fix hibernation, that's great.
You could start by making some pull request. I can assist you the first times with code/documentation clearness and consistency. If the feedback from the community is good, you (and others who'd like to) can take the lead of the project!

**Reply by mhl221135** [reply]
Ok, i try to update bootloader when it was published
Then if i have any questions i ask you about its and also share my results
But anyway thank you for your great gob!

**Comment by duszmox** [comment]
Hi, first of all thank you for your amazing work on this repo, you basically started my Hackintosh "carrier". I have taken the courage to continue the fork of this repo, which I started back when I released the first Big Sur EFI and guide, and already updated all the kexts and the bootloader to 0.6.8. Also added some new guides for example the "How to add Apple Watch authentication to sudo" one. I am planning to release some other cool stuff and continue this great work.

**Reply by mhl221135** [reply]
Its cool that you continue Simone job, i think we  all can make it better together )

**Reply by simprecicchiani** [reply]
Hey @duszmox, thank you for the support you gave in the past and sill give. My idea was for you to continue on this specific repo, where all issues already reside and discussion have taken place. If it's ok I'll make you and @mhl221135 collaborators, you will have full access to the repository management and donations can be splitted!

**Reply by duszmox** [reply]
Sure, that's a great idea, looking forward for working together @mhl221135!

**Reply by mhl221135** [reply]
> Hey @duszmox, thank you for the support you gave in the past and sill give. My idea was for you to continue on this specific repo, where all issues already reside and discussion have taken place. If it's ok I'll make you and @mhl221135 collaborators, you will have full access to the repository management and donations can be splitted!

I think we can make its together i have no much time but when there more peoples its will be faster )Thank for your great project

**Comment by nguyenthienhoan** [comment]
hello Simprecicchiani, i'm from Vietnamese may be my english not good. i know this repo about 5 months ago, last month OpenCore 0.6.7 was release but you still not update this and i was creat an issue from this repo for the question, hope you update this. i have T460s device too, and love ThinkPad. Now OpenCore 0.6.8 release, can you help me continue this repo. Thank you so much.

**Reply by mhl221135** [reply]
Hello, there is updated fork of this repo https://github.com/duszmox/Thinkpad-T460s-macOS-OpenCore you can use it now (i check all working flawlesly)
